api/models/comments-models.js

```diff
@@ -8,4 +9,23 @@ exports.selectCommentsByArticleId = (article_id)=> {
     .then(({ rows })=> {
         return rows;
     })
+}
+
+exports.insertCommentByArticleId = (newComment, article_id)=> {
+   const  { body, username }  = newComment;
+   const votes = 0;
+   const created_at = 'NOW()';
+   
```

Comment:
Votes and created_at aren't needed here. The seed sets votes to 0 and creates the created_at date for you.